### PR TITLE
Update news for 1.4.0

### PR DESCRIPTION
**Pull request by DjWarmonger**
With extra screenshots

### CODE REVIEW

File: content/news/vcmi-1.4.0-release.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
++++
+
+Another major realease just came out. As promised, this time we focused on multiplayer and quality-of-life features that make VCMI more accessible to new players and well as HoMM veterans. It’s not only about fixing bugs, but also cleaning up some unintuitive menus, options and descriptions to match player expectations.
+
+## Simultaneous Turns
+Long-awaited and challenging feature, which is also a key to wider adoption by PvP community. Few different modes are possible along with customizable turn timers.
+
+## Improved GUI widgets
+Large spellbook, hero slection dialog, more tooltips in adventure map, folders in map / save browser and more.
```
Comment (dydzio0614) [inline content/news/vcmi-1.4.0-release.md]: slection -> selection

**Comment by dydzio0614** [comment]
screenshot with turn options shows outdated version

File: content/news/vcmi-1.4.0-release.md (hunk 1 above)
Comment (Laserlicht) [inline content/news/vcmi-1.4.0-release.md:21]:
```suggestion
Large spellbook, hero selection dialog, more tooltips in adventure map, folders in map / save browser and more.
```

**Comment by DjWarmonger** [comment]
Replaced with screenshot I just took now

**Review by dydzio0614 — APPROVED** [review]

**Pull request merged**